docs/getting-started/fundamentals/static-dynamic-rendering.md

```diff
@@ -0,0 +1,166 @@
+
+### How to define static rendering for a block
+
+The `save` function, which can be defined when [registering a block on the client](https://developer.wordpress.org/block-editor/getting-started/fundamentals/registration-of-a-block/#registration-of-the-block-with-javascript-client-side), determines the markup of the block that will be stored in the database when the content is saved and eventually returned to the front end when there's a request. This markup is stored wrapped up in [unique block delimiters](https://developer.wordpress.org/block-editor/getting-started/fundamentals/markup-representation-block/) but only the markup inside these block indicators is returned as the markup to be rendered for the block on the front end.
+
+<details><summary><em>Example static rendering of <code>preformatted</code> core block</em></summary>
+<br/>
+For example, the following <a href="https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/preformatted/save.js"><code>save</code> function</a> of the <a hreh="https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/preformatted"><code>preformatted</code></a> core block...
+
+```js
+import { RichText, useBlockProps } from '@wordpress/block-editor';
+
+export default function save( { attributes } ) {
+	const { content } = attributes;
+
+	return (
+		<pre { ...useBlockProps.save() }>
+			<RichText.Content value={ content } />
+		</pre>
+	);
+}
+```
+
+...generates the following markup representation of the block when `attributes.content` has the value `"This is some preformatted text"`...
+
+```html
+<!-- wp:preformatted -->
+<pre class="wp-block-preformatted">This is some preformatted text</pre>
+<!-- /wp:preformatted -->
+```
+
+...and it will return the following markup for the block to the front end when there's a request.
+
+```html
+<pre class="wp-block-preformatted">This is some preformatted text</pre>
+```
+
+</details>
+
+<br/>
+
+Blocks with dynamic rendering can also define a markup representation of the block (via the `save` function) which can be processed in the server before returning the markup to the front end. If no dynamic rendering method is found, any markup representation of the block in the database will be returned to the front end.
+
+<div class="callout callout-info">
+The markup stored for a block can be modified before it gets rendered on the front end via hooks such as <a href="https://developer.wordpress.org/reference/functions/render_block/"><code>render_block</code></a> or via <code>$render_callback</code>.
+</div>
+
+Some examples of core blocks with static rendering (a.k.a. static blocks), this is blocks which output for the front end is statically generated when saved to the database (as returned by their `save` functions), are:
```

Comment:
This sentence is trying to explain several things, and it just becomes confusing to understand. I really don't think you need to re-explain what was already said. I'd just simply write:

> Some examples of core blocks with static rendering are:

Otherwise, it needs to be a couple of sentences, broken down into individual thoughts.